build/parseSpec.c

```diff
@@ -1109,6 +1123,25 @@ static rpmRC parseSpecSection(rpmSpec *specptr, int secondary)
 	    goto errxit;
 	}
 
+	if (stage == PARSE_GENERATED) {
+	    switch (parsePart) {
+	    case PART_PREP:
+	    case PART_BUILD:
+	    case PART_INSTALL:
+	    case PART_CHECK:
+	    case PART_CLEAN: /* ???? */
+	    case PART_BUILDARCHITECTURES:
+	    case PART_PATCHLIST:
+	    case PART_SOURCELIST:
+	    case PART_BUILDREQUIRES:
+	    case PART_CONF:
```

Comment:
Not sure if this was in the earlier version or I just missed it, but this info should go to struct PartRec for the same reason as the tags: to force people to consider the matter when adding new.

Reply:
Yes, it was. But I also was confused when seeing it and at first thought the rebase went wrong.